src/helpers.rs

```diff
@@ -616,6 +616,14 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
 
         Ok(wchars)
     }
+
+    fn in_std(&self) -> bool {
```

Comment:
The name is not very self-explaining... *what* is in std?
Please at least add a doc-comment. Maybe there's a more self-explaining name?

Reply:
This calls `this.frame()`... so `frame_in_std`?

Reply:
Yeah, makes sense. :)